.changeset/funny-bananas-deny.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/ssr': patch
```

Comment:
Should this be minor, since it's breaking?

Reply:
Re-worked such that it is no longer breaking.